scene/animation/animation_tree.cpp

```diff
@@ -919,20 +933,29 @@ void AnimationTree::_process_graph(float p_delta) {
 								t->scale = scale;
 							}
 
-							if (err != OK) {
+							if (err != OK) { // error happens when I animated an object transform manually?? fix
```

Comment:
```suggestion
							if (err != OK) { // Error happens when I animated an object transform manually?? fix.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).